Acs gridded enriched

abigailsnyder opened the pull request:
This branch

1. adds a reproducibility argument to the `make_recipes` helper function 
2. Adjusts `modules/tas_psl_stitching_paper1.py` so that it uses the bracketing scenarios only in the archive for experiments, stitches one realization per target ensemble for just the first five target ensembles, and some robustness tweaks.
3. Adjusts `modules/paper1_enrich_ensembles.py` but I haven't gotten there yet

kdorheim reviewed (approved):
The code looks good and based on a quick quality check (see [here](https://rpubs.com/kdorheim/884319)) the recipes are set up correctly, future stitched products look good based on the gsat.

abigailsnyder commented:
Ok 
3. Adjusts modules/paper1_enrich_ensembles.py: 
- general tidying
- setting the archive at the top of the script so it's easy to change between bracketing vs all experiments
- These experiments keep the target the same no matter what (the entire target ensemble) and it's just a facet of increasing the number of realizations in the archive (and what experiments are included in the archive).
- `*_compdata.csv` files contain the comparison data for the target realizations but only the N ensemble members specifically sampled for the archive, which is not necessarily the entire target
- added `_compdata_entire_target.csv` that is the comparison data for the entire target. 
- also added a `tol=0.075` case to the already included `0.07, 0.1, 0.13` for consistency

kdorheim reviewed (commented):
the changes to the paper1_enrich_ensemble.py look good!
